config_panel.toml

```diff
@@ -0,0 +1,209 @@
+version = "1.0"
+
+[homeserver]
+name = "Homeserver Privacy"
+services = ["__APP__"]
+
+    [homeserver.config]
+    name = "Configuration Options"
+
+            [homeserver.config.backup_before_upgrade]
+            ask = "Backup before upgrade"
+            type = "boolean"
+            yes = true
+            no = false
+            help = "Set to 'true' to backup the Synapse YunoHost App before each upgrade."
+
+            [homeserver.config.server_statistics]
+            ask = "Server statistics"
+            type = "boolean"
+            yes = true
+            no = false
+            help = "Set to 'true' to send anonymous statistics to Synapse to improve performance."
+            bind = "report_stats:/etc/matrix-__APP__/homeserver.yaml"
+
+    [homeserver.privacy]
+    name = "Data Privacy"
+
+            [homeserver.privacy.default_identity_server]
+            ask = "Identity server suggested to clients."
+            type = "url"
+            help = "By default, no suggestion is made, so it is left up to the client."
+            bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+            [homeserver.privacy.allow_public_rooms_without_auth]
+            ask = "Public rooms directory"
+            type = "boolean"
+            yes = true
+            no = false
+            help = "If set to 'false', requires authentication to access the server's public rooms directory through the client API and forbids any other homeserver to fetch the server's public rooms directory via federation."
+            bind = ":/etc/matrix-__APP__/homeserver.yaml"
+
+            [homeserver.privacy.allow_public_rooms_over_federation]
+            ask = "Public rooms directory"
+            type = "boolean"
+            yes = true
+            no = false
+            help = "If set to 'false', requires authentication to access the server's public rooms directory through the client API and forbids any other homeserver to fetch the server's public rooms directory via federation."
```

Comment:
Same `ask` and same `help` with the settings `allow_public_rooms_without_auth`.

Reply:
done